app/celer-g4/RunInputIO.json.cc

```diff
@@ -17,6 +17,25 @@
 #include "celeritas/field/FieldDriverOptionsIO.json.hh"
 #include "celeritas/phys/PrimaryGeneratorOptionsIO.json.hh"
 
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+void from_json(nlohmann::json const& j, TrackOrder& value)
```

Comment:
Since we use this in celer-sim as well, maybe we should make these in `celeritas/TypesIO.json.hh`? We could make it small and inline...

Reply:
Yeah good call.